torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -89,27 +89,32 @@ class FullyShardedDataParallel(nn.Module):
     A wrapper for sharding Module parameters across data parallel workers. This
     is inspired by `Xu et al.`_ as well as the ZeRO Stage 3 from DeepSpeed_.
     ``FullyShardedDataParallel`` is commonly shorten to FSDP.
+
     .. _`Xu et al.`: https://arxiv.org/abs/2004.13336
     .. _DeepSpeed: https://www.deepspeed.ai/
+
     Example::
         import torch
-        from torch.distributed._fsdp import FullyShardedDataParallel as FSDP
+        from torch.distributed.fsdp import FullyShardedDataParallel as FSDP
         torch.cuda.set_device(device_id)
         sharded_module = FSDP(my_module)
         optim = torch.optim.Adam(sharded_module.parameters(), lr=0.0001)
         x = sharded_module(x, y=3, z=torch.Tensor([1]))
         loss = x.sum()
         loss.backward()
         optim.step()
```

Comment:
Formatting is messed up in docs preview: https://docs-preview.pytorch.org/72122/fsdp.html?highlight=fsdp#module-torch.distributed.fsdp

I think you can add a new line after Example:: and it will fix it. Or add >>> to each line

Reply:
is this link for PT1.11 docs? 

I changed like this before: 

            Example::

                >>> def custom_auto_wrap_policy(
                >>>     module: nn.Module,
                >>>     recurse: bool,
                >>>     unwrapped_params: int,
                >>>     # These are customizable for this policy function.
                >>>     min_num_params: int = int(1e8),
                >>> ) -> bool:
                >>>     return unwrapped_params >= min_num_params

Reply:
Yes this is for 1.11, the link [https://docs-preview.pytorch.org/72122](https://docs-preview.pytorch.org/72122) shows the docs built from PR #72122